README.md

```diff
@@ -16,8 +16,13 @@ So far we tested Northstar on
 * 64-bit ARM
 * x86_64
 
-### Missing from implementation
+### Everything that is checked is implemented
```

Comment:
Why not just name this sections features?

Reply:
yeah....better